src/element/Hyperlink.tsx

```diff
@@ -69,6 +70,10 @@ export const Hyperlink = ({
 
     const link = normalizeLink(inputRef.current.value);
 
+    if (!element.link && link) {
+      trackEvent("hyperlink", "create");
+    }
```

Comment:
I moved the creation in here and only checking if we actually submit the form with non-empty link (from previously empty)